server/config.ts

```diff
@@ -104,8 +104,8 @@ export async function loadConfig(workingDir: string): Promise<Config> {
     Object.entries(env).forEach(([key, value]) => Deno.env.set(key, value))
   }
   config.plugins = [
+    cssLoader(), // add the css loader as default
```

Comment:
Maybe we could supply some option if people want it enabled or not, because some people may want to handle CSS in their own way.

Reply:
ok! i will remove it

Reply:
@ije I don't think it is necessary to import, maybe something like `config.defaultCSS ? cssLoader() : undefined` and make sure it skips over `undefined`.

Reply:
we can provide an option in `init` command to help add the css loader:

```ts
import css from '../../plugins/css.ts'
import type { Config } from '../../types.ts'

export default (): Config => ({
  plugins: [
    css({
      postcss: { plugins: ['autoprefixer'] }
    })
  ]
})
```